### ISSUE TITLE
Function definition does not match UML design

### ISSUE THREAD

**Issue by LodisAIE** [TerrickThib/RPGShop#2]
https://github.com/TerrickThib/RPGShop/blob/ef63b0fa6603b8e0abfb3c131148641afb794509/Game.cs#L158

This function was meant to return a list the list of items and the save and load options in an array. That's why its return type was a `string[]`. If you did it this way, you could pass the array that was returned into the `GetInput` function. Which would helped condense your code.

**Comment by TerrickThib** [comment]
Yes sir when i was first writing my code i didn't realize what the function was for so i di what ever with it. I later realized my mistake thank you for your feed back.
